<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6CGC2KB" failures="0" tests="1" name="com.medmantra.testcases.OPBilling.TC_019_BillExecution_DepositRefund_Card_Submit" time="164.848" errors="1" timestamp="2022-08-18T13:20:54 IST" skipped="0">
  <testcase classname="com.medmantra.testcases.OPBilling.TC_019_BillExecution_DepositRefund_Card_Submit" name="runDepositCardSubmit" time="164.848">
    <error message="See the reporter for details." type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: See the reporter for details.
at com.framework.utils.Reporter.reportStep(Reporter.java:92)
at com.framework.utils.Reporter.reportStep(Reporter.java:107)
at com.framework.selenium.api.base.SeleniumBase.takeSnap(SeleniumBase.java:913)
at com.framework.utils.Reporter.reportStep(Reporter.java:80)
at com.framework.utils.Reporter.reportStep(Reporter.java:107)
at com.medmantra.pages.CollectDepositPage.clickOnSubmit(CollectDepositPage.java:334)
at com.medmantra.testcases.OPBilling.TC_019_BillExecution_DepositRefund_Card_Submit.runDepositCardSubmit(TC_019_BillExecution_DepositRefund_Card_Submit.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runDepositCardSubmit -->
  <system-out/>
</testsuite> <!-- com.medmantra.testcases.OPBilling.TC_019_BillExecution_DepositRefund_Card_Submit -->
